content/news/playtest-20160424.md

```diff
@@ -0,0 +1,81 @@
+We are happy to present to you the second playtest in this release cycle!
+
+Many users &ndash; indeed more than we anticipated (over 2000!) &ndash; have tried out the first playtest and gave us a lot of feedback. Almost inevitably, a number of bugs were reported (thanks to all the testers who took their time to do so!).
+
+We have now fixed all of those issues, including:
+
+* Crashes when opening the in-game menu on some maps (Fort Lonestar, for example)
+* Broken left click orders
+* Too many infantry units coming out of sold buildings
+* Submarines not being targetable
+* Stealth units being targetable even when stealthed under some circumstances
+
+Additionally, the Red Alert mod has seen more refinements to its map pool: a few lower quality maps have been removed, while some new tournament-grade maps have been added.
+
+As usual, the complete list of changes can be seen [on our wiki](http://github.com/OpenRA/OpenRA/wiki/Changelog/4ad573f5615b1cdaebdd0d8c5a4c09bcb042798a).
+
+We promised in the last news post to detail the new map format and other map-related changes here. Skip to below the break for the full scoop.
+
+Installers for all supported operating systems are available from [the download page](/downloads). Of course we are still interested to hear about any bugs and issues you come across, so please don't hesitate to report them [on our bug tracker](http://bugs.openra.net), in the OpenRA [IRC channel](/community/), or here in the comments.
+
+At this point we'd like to address a topic already mentioned in the previous news post. In order to get a better understanding of the composition of our user base and the way our software is being used, we decided to add an anonymous, opt-out data gathering facility to OpenRA. When enabled, the game will send us a few technical details about your system.
+The pieces of information collected are these:
+
+* An anonymous ID (e.g. "5ba68aa1-1ee8-4d6d-b197-51681347f252")
+* Your OS version (e.g. "Windows: Microsoft Windows NT 6.1.7601 Service Pack 1" or "OSX: Unix 14.5.0.0")
+* Your .NET runtime version (e.g. ".NET CLR 4.0.30319.42000" or "Mono 4.0.2 ((detached/c99aa0c) CLR 4.0.30319.17020")
+* Your GL driver version (e.g. "4.5.0 NVIDIA 352.63" or "3.0 Mesa 10.5.9")
+* Your OS language setting (e.g. "en" or "ru")
+* The OpenRA version / mod / modversion that you launched
+
+As already mentioned, this data is sent to us anonymously, using a randomly generated ID. If you are uncomfortable with this, you can disable this functionality by unchecking the "Send System Information" checkbox in the advanced settings dialog, manually editing your `settings.yaml` file, or starting the game using the `Debug.SendSystemInformation=false` argument.
```

Comment:
I think we can now remove "As already mentioned, this data is sent to us anonymously, using a randomly generated ID", since that is clearly stated just above.